src/main/java/com/avispl/symphony/dal/avdevices/mixer/vaddio/easyip/EasyIPMixerCommunicator.java

```diff
@@ -445,21 +491,21 @@ public void controlProperty(ControllableProperty controllableProperty) throws Ex
 						indexCamera = EasyIPMixerMapping.getValueByName(group.replace(EasyIPMixerConstant.VIDEO_INPUT, EasyIPMixerConstant.EMPTY));
 						value = checkValidInput(1, 20, value);
 						command = propertyItem.getControlCommand().replace("$1", indexCamera).replace("$2", value);
-						sendCommandToControlDevice(command, value, propertyKey);
+						sendCommandToControlDeviceWithExecutor(command, value, propertyKey);
 						updateCachedDeviceData(cacheKeyAndValue, property, value);
 						break;
 					case CAMERA_PAN:
 						indexCamera = EasyIPMixerMapping.getValueByName(group.replace(EasyIPMixerConstant.VIDEO_INPUT, EasyIPMixerConstant.EMPTY));
-						value = checkValidInput(-150f, 150f, value);
+						value = checkValidInput(-156.3, 151.7, value);
 						command = propertyItem.getControlCommand().replace("$1", indexCamera).replace("$2", value);
-						sendCommandToControlDevice(command, value, propertyKey);
+						sendCommandToControlDeviceWithExecutor(command, value, propertyKey);
 						updateCachedDeviceData(cacheKeyAndValue, property, value);
 						break;
 					case CAMERA_TILT:
 						indexCamera = EasyIPMixerMapping.getValueByName(group.replace(EasyIPMixerConstant.VIDEO_INPUT, EasyIPMixerConstant.EMPTY));
-						value = checkValidInput(-30, 90, value);
+						value = checkValidInput(-30, 92.5, value);
```

Comment:
Also these, is there any chance they will change in future or for other devices?